lv2/log/logger.h

```diff
@@ -56,7 +56,7 @@ typedef struct {
    to the log's print functions.
 */
 static inline void
-lv2_log_logger_set_map(LV2_Log_Logger* logger, LV2_URID_Map* map)
+lv2_log_logger_set_map(LV2_Log_Logger* logger, const LV2_URID_Map* map)
```

Comment:
Similarly to my other comment, this can technically be const, but calling `map` really isn't "constey".